sktime/transformations/compose/_column.py

```diff
@@ -192,6 +193,9 @@ def __init__(self, transformers, remainder=None, feature_names_out="auto"):
             self._anytagis_then_set(
                 "capability:missing_values:removes", False, True, l_transformers
             )
+            self._anytagis_then_set(
+                "capability:categorical_in_X", True, False, l_transformers
```

Comment:
should it not be the other way round? If at least one component can *not* handle categoricals, then the composite can *not* hanlde categoricals, i.e., `False, True`?

Reply:
No it's the column ensemble transformer where components are fed different input columns. So even if one of the components cannot handle categorical, we cannot say that the composite cannot handle as well.